### PR TITLE
Hide guest registration if disabled

### PR DESCRIPTION
**Pull request by budokaiman**
Addresses issue #494

### CODE REVIEW

File: ui/src/pages/user/Register.svelte [hunk 1]
```diff
@@ -117,6 +118,12 @@
       });
   }
 
+  onMount(() => {
+    if ($user.id) {
```
Comment (StevenWeathers) [inline ui/src/pages/user/Register.svelte:122]: this would prevent guest users from registering into a full account

Comment (budokaiman) [reply ui/src/pages/user/Register.svelte]: Ah, that make sense. I've put the user ID check back the way it was.

**Review by StevenWeathers — DISMISSED** [review]

File: ui/src/pages/user/Register.svelte [hunk 2]
```diff
@@ -1,4 +1,5 @@
 <script lang="ts">
+  import { onMount } from 'svelte';
```
Comment (scottux) [inline ui/src/pages/user/Register.svelte:2]: This looks like a leftover from a refactor.

**Review by scottux — CHANGES_REQUESTED** [review]
Please remove the unused onMount import.

**Review by scottux — APPROVED** [review]

**Review by StevenWeathers — APPROVED** [review]

**Pull request merged**